commercetools-models/src/main/java/io/sphere/sdk/customers/commands/CustomerInStoreSignInCommand.java

```diff
@@ -28,11 +37,12 @@
 
     private final String storeKey;
 
-    private CustomerInStoreSignInCommand(final String storeKey, final String email, final String password, @Nullable final String anonymousCartId, @Nullable final String anonymousId, final AnonymousCartSignInMode anonymousCartSignInMode,
+    private CustomerInStoreSignInCommand(final String storeKey, final String email, final String password, @Nullable ResourceIdentifier<Cart> anonymousCart, @Nullable final String anonymousCartId, @Nullable final String anonymousId, final AnonymousCartSignInMode anonymousCartSignInMode,
```

Comment:
```suggestion
    private CustomerInStoreSignInCommand(final String storeKey, final String email, final String password, @Nullable ResourceIdentifier<Cart> anonymousCart, @Nullable final String anonymousId, final AnonymousCartSignInMode anonymousCartSignInMode,
```

As this is a private constructor you can change the signature and completely drop the anonymousCartId. Now you have to refactor the other methods to create a ResourceIdentifier if needed.

Reply:
ok thanks I'm gonna change it